.travis/install.sh

```diff
@@ -50,3 +50,15 @@ pip install -r requirements.d/development.txt
 pip install codecov
 python setup.py --version
 pip install -e .[fuse]
+
+if [ -d "$HOME/.rustup/toolchains" ] && [ -f "$HOME/.cargo/env" ]; then
+    source ~/.cargo/env
+    rustup toolchain install stable # also updates
+    rustup default stable
+else
+    curl https://sh.rustup.rs -sSf | sh -s -- -y
+    source ~/.cargo/env
+fi
+
+cd test-wrapper
+cargo build --release
```

Comment:
as long as it is not stable / not in Vagrantfile, I just need something that works, so rustup is it.